### PR TITLE
[CPU] General fixes for dynamic shapes. Part 3

### PR DESCRIPTION
**Pull request by mandrono**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *74316*

### CODE REVIEW

**Comment by mandrono** [comment]
fix error message for `shapeInfer`

**Comment by mandrono** [comment]
fix: https://github.com/openvinotoolkit/openvino/pull/8597#discussion_r775275539

File: src/plugins/intel_cpu/src/nodes/mkldnn_eltwise_node.cpp [hunk 1]
```diff
@@ -1698,7 +1698,7 @@ void MKLDNNEltwiseNode::execute(mkldnn::stream strm) {
 
         execPtr->exec(*this, args_ptrs, dims_out);
     } else {
-        IE_THROW() << "Can't execute eltwise node. Primitive didn't created";
+        IE_THROW() << "Can't execute eltwise node with name: " << getName() << ". Primitive haven't created";
```
Comment (dmitry-gorokhov) [inline src/plugins/intel_cpu/src/nodes/mkldnn_eltwise_node.cpp:1701]: `hasn't` or even better `isn't`

File: src/plugins/intel_cpu/src/memory_desc/cpu_blocked_memory_desc.cpp [hunk 2]
```diff
@@ -112,8 +112,9 @@ bool CpuBlockedMemoryDesc::canComputeMemSizeZeroDims() const {
 }
 
 size_t CpuBlockedMemoryDesc::getCurrentMemSizeImp() const {
-    int64_t e_size = getOffsetPadding() + 1;  // size in bytes (from begin of data to last element)
+    int64_t e_size = getOffsetPadding();  // size in bytes (from begin of data to last element)
     if (!getShape().hasZeroDims()) {
+        e_size += 1;
```
Comment (dmitry-gorokhov) [inline src/plugins/intel_cpu/src/memory_desc/cpu_blocked_memory_desc.cpp:117]: As we discussed offline: this behavior should be fixed in separate PR

**Review by dmitry-gorokhov — APPROVED** [review]
LGTM. Please rebase on latest master

File: src/plugins/intel_cpu/src/memory_desc/cpu_blocked_memory_desc.cpp (hunk 2 above)
Comment (mandrono) [reply src/plugins/intel_cpu/src/memory_desc/cpu_blocked_memory_desc.cpp:117]: will be done in PR: https://github.com/openvinotoolkit/openvino/pull/9466
